docs/continuous-delivery/gitops/secret-management/secret-management-gitops.md

```diff
@@ -0,0 +1,177 @@
+# Secret Management with GitOps
+
+GitOps primarily have git repositories in the center considered a source of truth for managing both infrastructure and application. This Infrastructure and application will require secured access to other resources of the system through secrets. Committing clear text secrets into git repositories is unacceptable even if the repositories are private to your team and organization. Teams need a secure way to handle secrets when using GitOps.
```

Comment:
A few suggestions to consider:
* GitOps primarily --> GitOps <something> (projects | solutions | etc?) primarily... (also, is the "primarily" qualifier needed?)
* "This Infrastructure" --> lower-case infrastructure
* "clear text" --> "clear-text". 
* Consider expanding briefly on why committing secrets into repositories is a "bad thing™️"

Reply:
Should I clarify why committing clear texts secrets is a bad practice or commiting secrets in general?

Reply:
I chose to believe you meant the former and tried my best in providing an example. Let me know if you'd word it differently 😄